lua/lspconfig/configs/autohotkey_lsp.lua

```diff
@@ -0,0 +1,56 @@
+-- NOTE: AutoHotkey is used only on windows
+local function get_autohotkey_path()
+  local path = vim.fn.exepath('autohotkey.exe')
+  return #path > 0 and path or ""
+end
+
+return {
+  default_config = {
+    cmd = { 'autohotkey_lsp', '--stdio' },
+    filetypes = { 'autohotkey', 'ahk', 'ah2' },
```

Comment:
'ahk', 'ah2' look like extensions, not filetypes. `filetypes` matches nvim "filetype" kinds, not extensions.

```suggestion
    filetypes = { 'autohotkey' },
```

Reply:
Ok, removed :)